pytorch/Dockerfile

```diff
@@ -91,15 +89,51 @@ COPY multinode-requirements.txt .
 
 RUN python -m pip install --no-cache-dir -r multinode-requirements.txt
 
+ENV LD_LIBRARY_PATH="/lib/x86_64-linux-gnu:${LD_LIBRARY_PATH}:/usr/local/lib/python${PYTHON_VERSION}/dist-packages/oneccl_bindings_for_pytorch/opt/mpi/libfabric/lib:/usr/local/lib/python${PYTHON_VERSION}/dist-packages/oneccl_bindings_for_pytorch/lib"
```

Comment:
These are the updates from Tyler's SSH PR, right? Are they just here for testing both changes together? Are we assuming that Tyler's PR will merge first?

Reply:
I grabbed Tyler's PR into this, so yeah this will go after Tyler's